01week/datatypes.js

```diff
@@ -0,0 +1,46 @@
+//display the current day and time
+const today = new Date();
+const dd = today.getDate();
+const mm = today.getMonth();
+const yyyy = today.getFullYear();
+Date(dd,mm,yyyy)
+
+//convert a number to a string
+var num = 15;
+var n = num.toString();
```

Comment:
This is a variable. Also never use var.  I need a function to turn any number into a string and return that string.